src/component/base/methods.js

```diff
@@ -51,6 +52,39 @@ export default {
     enumerable: true,
     configurable: false,
   },
+  $bubbleInput: {
+    /**
+     * Bubble up a keyboard event to a parent component without changing focus
+     * @this {import('../../component').BlitsComponent}
+     * @param {string} key - The key name (e.g., 'enter', 'back', 'up')
```

Comment:
is this needed as a separate param? why can't we just pass the `event` in the same way we do with `$focus()` ?